jvector-base/src/main/java/io/github/jbellis/jvector/graph/disk/FusedADCNeighbors.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.github.jbellis.jvector.graph.disk;
+
+import io.github.jbellis.jvector.vector.types.ByteSequence;
+
+public interface FusedADCNeighbors {
```

Comment:
there is only one implementation, do we need the extra interface?

Reply:
looks like this is for consistency w/ some of the others in which case I'm agnostic